### PR TITLE
add meanworks

### PR DESCRIPTION
**Pull request by azat-co**
add meanworks: automated workshop on MEAN stack with angular full stack generator and series of videos to give an overview of the MEAN stack and the workshop tool

- Tool: https://github.com/azat-co/meanworks
- Playlist: https://www.youtube.com/playlist?list=PLguYmmjtxbWEdBy8FqGcz7ZIdl0cHV9SY
- Slides: https://github.com/azat-co/meanworks/blob/master/slides/README.pdf

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -305,6 +305,11 @@ <h3 data-i18n="index-workshoppers-elective-header">Electives</h3>
           <p data-i18n="workshopper-pattern-lab-workshop">Learn the basics of Pattern Lab.</p>
           <code>npm install -g pattern-lab-workshop</code>
         </div>
+        <div id="meanworks" class="workshopper">
+          <h4><a class="js-workshop-link" href="https://github.com/azat-co/meanworks" target="_blank">MEANWorks</a></h4>
+          <p data-i18n="workshopper-node-debug-school">M.E.A.N: MongoDB/Mongoose, Express, Angular and Node Workshop with Angular Fullstack Generator</p>
```
Comment (martinheidegger) [inline index.html]: Please use a new data-i18n key.

Comment (azat-co) [inline index.html]: @martinheidegger Do you mean in the `<p>`? I have this:

```
<p data-i18n="workshopper-node-debug-school">M.E.A.N...
```

Or do you mean else where?

Comment (martinheidegger) [inline index.html]: @azat-co Yes, it says `workshopper-node-debug-school`. It should be something like `workshopper-meanworks`

Comment (azat-co) [inline index.html]: @martinheidegger my bad, I copied the structure from debug and didn't update that field... let me push changes.

**Comment by azat-co** [comment]
@martinheidegger okay, I updated the i18n attribute :laughing:

**Comment by martinheidegger** [comment]
Merged

**Comment by azat-co** [comment]
@martinheidegger sweet. thank you!

**Pull request merged**